src/Socket.cpp

```diff
@@ -237,30 +229,33 @@ void Socket::sendMessage(MessagePtr message)
     d->sendQueue.push_back(message);
 }
 
-MessagePtr Socket::takeNextMessage()
+MessagePtr Socket::takeNextMessage(bool blocking)
 {
     // Set 'listener' in wait mode until a new message received
```

Comment:
This comment has become outdated, the actual `wait(...)` now takes place at the end of the method.